analyzer/tests/unit/test_skiplist.py

```diff
@@ -0,0 +1,67 @@
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+"""Skip list tests."""
```

Comment:
"""
Skip list tests.

NOTE: There is a similar file in the web part.
Please consider to update that in case of changing this.
"""